packages/ui/src/components/timeline/Label.tsx

```diff
@@ -59,6 +61,9 @@ export function Label({event, scene}: LabelProps) {
             handleDrag(e);
           }
         }}
+        onMouseLeave={() =>
+          (labelClipDragging.value = {left: 0, dragging: false})
+        }
```

Comment:
The second callback passed to `useDrag` is called when the dragging ends. That's when this signal should be reset.

Reply:
When i use reset the value in the second callback there seems to be a bug when I leave the mouse click and not drag the mouse the `signal always becomes true` 

Also the `pixelsToFrames` and `secondsToFrame` methods are calculating the distance according to the timeline visible in the viewport making the vertical line appear in different positions

Reply:
**When not dragging and leaving the mouse click**

https://github.com/motion-canvas/motion-canvas/assets/42911859/cfa81ce2-2d6d-4b59-8658-e688117f7c8f


**PixelsToFrames and SecondsToFrame methods**

https://github.com/motion-canvas/motion-canvas/assets/42911859/ca78d682-3232-4984-8876-f05589f685a8

Reply:
`framesToPixels` calculates the absolute position on the timeline, not related to the view.
You just need to account for the padding on the left caused by negative frames:
```ts
const position = state.framesToPixels(
  player.status.secondsToFrames(labelClipDragging.value.left),
) + sizes.paddingLeft;
```
Note how the time of the event is expressed in seconds, so you need to first call `secondsToFrames` to get the frames and *then* `framesToPixels` to turn those frames into seconds.

Reply:
Seems that there's a bug with `useDrag` that causes the first callback to be invoked one more time, right after the drag has finished.
The best approach would be to get rid of `useDrag` altogether (since it's a really hacky solution) and use pointer capture instead. This is something we do in newer components, you can find an example here: 
https://github.com/motion-canvas/motion-canvas/blob/main/packages/ui/src/components/layout/ResizeableLayout.tsx#L51-L67